digitaltwin/modelrepo/Authentication/ModelSharedAccessSignature.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+namespace Microsoft.Azure.Devices.Common.Authorization
+{
+    public class ModelSharedAccessSignature : SharedAccessSignature
+    {
+        internal string _repositoryId;
+
+        private ModelSharedAccessSignature(string shareAccessSignatureName, DateTime expiresOn, string expiry, string keyName, string signature, string encodedAudience, string repositoryId) : base(shareAccessSignatureName, expiresOn, expiry, keyName, signature, encodedAudience)
```

Comment:
When methods (even ctors) parameters get so numerous, it is best to give one per line:

```
private ModelSharedAccessSignature(
    string shareAccessSignatureName,
    DateTime expiresOn,
    string expiry,
    string keyName,
    string signature,
    string encodedAudience,
    string repositoryId)
    : base(
        shareAccessSignatureName,
        expiresOn,
        expiry,
        keyName,
        signature, encodedAudience)
```